package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@chevrotain/quick-start",
+  "description": "Your easy path to define a parser for your language based on Chevrotain DSL toolkit",
+  "version": "0.0.1",
+  "private": true,
+  "keywords": [
+    "chevrotain",
+    "starter pack",
+    "dsl",
+    "parser"
+  ],
+  "author": "Santhosh Kumar <brsanthu@gmail.com>",
```

Comment:
you may not want to publish your email publicly, your choice..

Reply:
Will remove.